data/synth_timing.xdc

```diff
@@ -798,6 +850,18 @@ set_multicycle_path        $ah_spi_mulcycs       -setup -start -to [get_ports {a
 set_multicycle_path [expr {$ah_spi_mulcycs - 1}] -hold  -start -to [get_ports {ah_tmpio11 ah_tmpio10}] ;# out
 set_multicycle_path        $ah_spi_mulcycs       -setup -end -from [get_ports ah_tmpio12] ;# in
 set_multicycle_path [expr {$ah_spi_mulcycs - 1}] -hold  -end -from [get_ports ah_tmpio12] ;# in
+## PMOD 0 SPI - use same methodology as Arduino SPI
+set pmod0_spi_mulcycs $ah_spi_mulcycs
```

Comment:
Nit: maybe ah_spi_mulcycs should be caused external header cycles.

Reply:
Could do, although I use the same 'name based on first block' approach for some other interfaces too. Would you like me to rename them as well?

Reply:
I might as well

Reply:
Thanks